extension.toml

```diff
@@ -0,0 +1,16 @@
+id = "gdscript"
+name = "GDScript"
+version = "0.0.3"
```

Comment:
Assuming Zed is intending to follow semver, this should be `0.1.0` now 🤔

Reply:
I'll bump the version